test/JDBC/expected/BABEL-2571.out

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
Let's try to avoid using `BABEL-XXX` in test file names. I know we have tons of such files but let's make it better at least for new test files. A better test file name could be `BABEL-guest` under  `ownership` folder.

Reply:
ive fixed this one. thanks